app/services/suggester/users/recent.rb

```diff
@@ -55,15 +55,21 @@ def recent_commenters(num_comments = 2, limit = 8)
       end
 
       def established_user_article_count
-        Rails.env.production? ? 4 : -1
+        Rails.cache.fetch("established_user_article_count", expires_in: 1.day) do
+          User.average(:articles_count)
```

Comment:
this is going to change significantly, as we go from 4 to 0.87, at least for DEV

Reply:
This is a bit of a bigger change from 4 to 1, what if we played with the ordering to ensure the ones with higher counts were near the top in line 47 and 54?

Reply:
not sure but yeah, we can play with the numbers until we get it right

cc @benhalpern @vaidehijoshi

Reply:
Ok, I let this sit for a bit and came up with this idea. What do you think of limiting those we take the average from to the ones with counts greater than 0 and then article scores greater than or equal to 0?
```ruby
irb(main):005:0> User.where("articles_count > 0").average(:articles_count).to_i
=> 7
irb(main):007:0> User.where("comments_count > 0").average(:comments_count).to_i
=> 5
irb(main):009:0> Article.where("score >= 0").average(:score).to_i
=> 16
```

Reply:
> limiting those we take the average from to the ones with counts greater than 0 and then article scores greater than or equal to 0?

@mstruve I like this idea, it also feels more in line with what a Forem creator would want as well

Reply:
Updated and ready for rereview! Thanks for all the feedback!